python-meh.spec

```diff
@@ -70,79 +44,60 @@ Requires: libreport-cli >= %{libreportver}
 Requires: libreport-python3 >= %{libreportver}
 Requires: python3-six
 
-%description -n python3-meh
+%description -n python3-%{srcname}
 The python3-meh package is a python 3 library for handling, saving, and reporting
 exceptions.
 
-%package -n python3-meh-gui
+%package -n python3-%{srcname}-gui
 Summary: Graphical user interface for the python3-meh library
-Requires: python3-meh = %{version}-%{release}
+Requires: python3-%{srcname} = %{version}-%{release}
 Requires: python3-gobject, gtk3
 Requires: libreport-gtk >= %{libreportver}
 
-%description -n python3-meh-gui
+%description -n python3-%{srcname}-gui
 The python3-meh-gui package provides a GUI for the python3-meh library.
 
-%endif
-
 %prep
 %setup -q
 
-%if 0%{?with_python3}
 rm -rf %{py3dir}
 cp -a . %{py3dir}
-%endif # with_python3
 
 %build
 make
 
-%if 0%{?with_python3}
 pushd %{py3dir}
 make PYTHON=%{__python3}
```

Comment:
PYTHON is python3, can you just get rid of this to make it more readable?

Reply:
Also, you are running `make` twice.

Reply:
Sure, removed.